### ISSUE TITLE
Error: Cannot destructure property `compile` of 'undefined' or 'null'

### ISSUE THREAD

**Issue by sahilsharma011** [lauthieb/code-notes#92]
**Describe the bug**
I'm unable to run `npm run dev` and dev server is not working on localhost:9080

**To Reproduce**
1. Git clone
2. npm install
3. npm run dev

**Expected behavior**
Local server running on localhost:9080

**Error Message**
```
TypeError: Cannot destructure property `compile` of 'undefined' or 'null'.
    at addHooks (/var/www/html/code-notes/node_modules/webpack-dev-server/lib/Server.js:115:49)
    at new Server (/var/www/html/code-notes/node_modules/webpack-dev-server/lib/Server.js:128:5)
    at Promise (/var/www/html/code-notes/.electron-vue/dev-runner.js:62:20)
    at new Promise (<anonymous>)
    at startRenderer (/var/www/html/code-notes/.electron-vue/dev-runner.js:42:10)
    at init (/var/www/html/code-notes/.electron-vue/dev-runner.js:169:16)
    at Object.<anonymous> (/var/www/html/code-notes/.electron-vue/dev-runner.js:178:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

**Additional context**
On searching a bit, I'm seeing that webpack-dev-server@3.1.11 requires a peer of webpack@^4.0.0, but we have "webpack": "^3.5.2" in our package.json. 
Also the I'm getting `Version: webpack 3.10.0` in my `npm run dev` output.

**Comment by jack-chapman** [comment]
I believe this problem was caused by Dependabot updating webpack-dev-server from 2.11.0 to ^3.1.11 in #87 

I rolled back my webpack-dev-server version to 2.11.0 and the dev server started up properly. Not an ideal solution, but it'll get you up and running.

**Comment by jack-chapman** [comment]
As a follow up, I'm having a look at upgrading this to use the newest version of electron-vue. @lauthieb Are you happy for me to do this? It would be a pretty big pull request I think 😄

**Comment by lauthieb** [comment]
Closed with #93, thanks @jack-chapman :tada:
